esmf-semantic-aspect-meta-model/src/test/java/org/eclipse/esmf/samm/TimeSeriesTest.java

```diff
@@ -0,0 +1,13 @@
+package org.eclipse.esmf.samm;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class TimeSeriesTest extends AbstractShapeTest {
+
+    @ParameterizedTest
+    @MethodSource( value = "versionsStartingWith2_0_0")
+    void testTimeSeriesValidationExtendedEntity (final KnownVersion metaModelVersion) {
+        checkValidity("time-series", "TimeSeriesTestEntity", metaModelVersion);
+    }
+}
```

Comment:
In addition to the positive test, we also need a negative example (which triggers the error).